doc/nep1.rst

```diff
@@ -301,3 +301,13 @@ Miscellaneous
     let a = """foo
     bar
     """
+
+- A getter API for a private field `foo` should preferably be named `foo`, not `getFoo`.
+  A getter-like API should preferably be named `getFoo`, not `foo` if:
+    * the API has side effects
+    * or the cost is not `O(1)`
+  For in between cases, there is no clear guideline.
+  In all cases (`foo` or `getFoo`), the 1st argument should be called `self`.
```

Comment:
No. Hardly any code does it via `self` and it's not important.

Reply:
i've removed this line from this PR to unblock it